language/move-prover/scripts/README.md

```diff
@@ -1,32 +1,39 @@
 There are three experimental scripts in this folder.
-1. `install-dotnet.sh` installs Dotnet Core.
-2. `install-boogie.sh` installs the latest version of Boogie from nuget.org.
-3. `install-z3.sh` installs a specific version of Z3.
+- `install-dotnet.sh` installs Dotnet Core.
+- `install-boogie.sh` installs a specific version of Boogie from nuget.org.
+- `install-z3.sh` installs a specific version of Z3.
 
-`install-dotnet.sh` must be executed before `install-boogie.sh`.  There is no other dependency.
-`install-dotnet.sh` installs the Dotnet Core SDK and the `dotnet` tool.  The installation could fail
-on your machine because a few directories must be writable by your user account.  You should
-change the ownership of these directories to your user.
-  sudo chown -R $(whoami) /usr/local/bin /usr/local/lib /usr/local/sbin
-And make sure that your user has write permission.
-  chmod u+w /usr/local/bin /usr/local/lib /usr/local/sbin
+# Installation steps
 
-The commands `boogie` and `z3` should be in your path once these scripts have finished successfully.
+1. The installation of Dotnet Core could fail on your machine because a few directories
+   must be writable by your user account.  You should change the ownership of these
+   directories to your user.
+   - sudo chown -R $(whoami) /usr/local/bin /usr/local/lib /usr/local/sbin
+   Then make sure that your user has write permission.
+   - chmod u+w /usr/local/bin /usr/local/lib /usr/local/sbin
+2. ./install-dotnet.sh
+3. ./install-boogie.sh
+4. ./install-z3.sh
 
-To check the location of the tools installed, use the following commands:
+Troubleshooting:
+On bash, the commands `boogie` and `z3` should be in your path once these scripts
+have finished successfully.  On zsh, you may have to add /Users/$(whoami)/.dotnet/tools
+manually to $PATH.
+
+# Running Move Prover
```

Comment:
I don't know what a better title would be but this section doesn't really teach me how to run Move Prover.

Reply:
Environment variable setting?

Reply:
Done. Take a look.